modules/seqexec/web/shared/src/main/scala/seqexec/web/model/boopickle/ModelBooPicklers.scala

```diff
@@ -30,13 +31,28 @@ trait ModelBooPicklers extends GemModelBooPicklers {
   // IMPORTANT The order of the picklers is very relevant to the generated size
   // add them with care
   //**********************
-  implicit val instrumentPickler = generatePickler[Instrument]
+  implicit val instrumentPickler = transformPickler(
+    (t: Int) =>
+      Instrument.all
+        .find(_.ordinal === t)
+        .getOrElse(throw new RuntimeException("Failed to decode instrument")))(
```

Comment:
It seems like we could define `enumPickler[A: Enumerated]` and factor out the repetition here and below. Can we add the ordinal to the error message?

Reply:
It also might be worth making a map or array or something so the lookup isn't linear with `.find` … I don't know how much of a bottleneck this is.

Reply:
Yeah, though these are not `Enumerated` yet